src/components/organisms/search-overview/search-overview.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import './search-overview.scss';
+
+export default ({
```

Comment:
Im ersten Schritt eine recht simple Komponente zur Ausgabe von globalen Suchergebnissen. Momentan wird nur ein simpler Loading-Indikatior ausgegeben und die Suchergebnisse in "stringified"er Version.

Reply:
Nicht mehr aktuell, da Suchergebnisse mittlerweile auf simple Art und Weise ausgegeben werden (nicht mehr stringified).